utils/python/CIME/case_st_archive.py

```diff
@@ -286,6 +286,10 @@ def _archive_restarts(case, archive, archive_entry,
                     else:
                         srcfile = os.path.join(rundir, restfile)
                         logger.info("removing interim restart file %s" %srcfile)
+                        try:
+                            os.remove(srcfile)
+                        except OSError as e:
+                            logger.warn("interim restart file %s does not exist " %srcfile)
```

Comment:
No, because I removed the "if (os.path.isfile(srcfile)):" outside of the try block because it will 
throw an exception if the file doesn't exist and so the "if" check was redundant.

I added the images into this branch so I can test the wiki doc links. Do you want a separate branch for these?